server/protocols/webadmin/webadmin-mailrepository/src/main/java/org/apache/james/webadmin/dto/ExtendedMailRepositoryResponse.java

```diff
@@ -19,28 +19,17 @@
 
 package org.apache.james.webadmin.dto;
 
-import java.io.UnsupportedEncodingException;
-
 import org.apache.james.mailrepository.api.MailRepositoryPath;
 
-public class ExtendedMailRepositoryResponse implements MailRepositoryResponse {
+public class ExtendedMailRepositoryResponse extends MailRepositoryResponse {
```

Comment:
I'm globally against this "extends" and would prefer you to revert this specific change.